src/main/java/eutros/multiblocktweaker/crafttweaker/gtwrap/impl/MCControllerTile.java

```diff
@@ -113,7 +113,8 @@ public List<IIMultiblockPart> getMultiblockParts() {
 
     @Override
     public boolean canShare() {
-        return getInternal().canShare();
+        //2.4.0 GTCEu changed sharing to be defined by the part
+        return true;
     }
```

Comment:
This method was removed entirely in CEu 2.4.0, so it can be removed here and from its super class as well